backend/core/src/seeder/seeds/listings/listing-triton-seed.ts

```diff
@@ -22,7 +22,7 @@ const tritonListing: ListingSeedType = {
   amenities: "Gym, Clubhouse, Business Lounge, View Lounge, Pool, Spa",
   buildingAddress: {
     city: "Foster City",
-    county: "San Mateo",
+    county: "SanMateo",
```

Comment:
I don't think this is the right approach -- IMO we should just store it as a raw human readable string to be consistent with the rest of the buildingAddress object. If we want to translate this in the future we can map human readable strings to keys. In the meantime let's just not `t()` this field when its shown on the partners portal to avoid the bug you're seeing.